model/task_lifecycle.go

```diff
@@ -972,14 +985,17 @@ func getVersionStatus(builds []build.Build) string {
 		return evergreen.VersionCreated
 	}
 
-	// started but not finished
+	// Check if builds are started but not finished.
 	for _, b := range builds {
+		if b.AllTasksBlocked {
```

Comment:
I think instead of this continue, line 993 could just also have`&& !b.AllTasksBlocked`

Reply:
sorry i feel like it makes so much sense when i look back on it :(

Reply:
No worries! It's easier as a reviewer, and this is more a style thing than a correctness thing